test/mpi/spawn/join.c

```diff
@@ -88,6 +88,8 @@ int main(int argc, char *argv[])
         MPI_Send(hostname, namelen + 1, MPI_CHAR, 0, 0, MPI_COMM_WORLD);
         MPI_Send(&port, 1, MPI_INT, 0, 1, MPI_COMM_WORLD);
 
+        MPI_Barrier(MPI_COMM_WORLD);
```

Comment:
Ok, that clarifies, but I don't think changing the test is the right thing here. We should prove that OFI is the problem and ask them to fix the underlying issue. Let's drop this patch.